Celeste.Mod.mm/Patches/Monocle/Tracker.cs

```diff
@@ -123,5 +123,109 @@ private static List<Type> GetSubclasses(Type type) {
             // don't hold references to all the types anymore
             _temporaryAllTypes = null;
         }
+
+        public static void AddTypeToTracker(Type type, Type trackedAs = null, bool inheritAll = false) {
+            AddTypeToTracker(type, trackedAs, inheritAll ? GetSubclasses(type).ToArray() : Array.Empty<Type>());
+        }
+
+        public static void AddTypeToTracker(Type type, Type trackedAs = null, params Type[] subtypes) {
```

Comment:
This method seems like a copy-paste of the everest patch for Initialize, would be nice to rewrite that patch to use this method, to not have so much duplicate logic that then needs to be updated twice if new tracker features get added.

Reply:
So, do you want me to have the Initialize method use this AddTypeToTracker method?